New snapshot configs (#5817)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/clickhouse-configs.md

</File>

</VersionBlock>

<VersionBlock firstVersion="1.9">

<File name='snapshots/<model_name>.sql'>

```jinja
{{
   config(

>      schema = "<schema_name>",

Reviewer: ```suggestion
     schema = "<schema-name>",
```

     unique_key = "<column-name>",
     strategy = "<strategy>",

>      updated_at = "<unpdated_at_column-name>",

Reviewer: ```suggestion
     updated_at = "<updated-at-column-name>",
```

   )
}}
```

</File>

</VersionBlock>

#### Snapshot Configuration

> | Option          | Description                                                                                                               | Required?                                                                            |

Reviewer: I don't think we need this table? It seems duplicative of the other table here -> https://docs.getdbt.com/docs/build/snapshots#snapshot-configurations

Author: Sure, can remove and link to snapshots page instead!

|-----------------|---------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|
| <VersionBlock lastVersion="1.8"> `target_schema` </VersionBlock> <VersionBlock firstVersion="1.9"> `schema` </VersionBlock>| A ClickHouse's database name where the snapshot table will be created.       | <VersionBlock lastVersion="1.8"> Required </VersionBlock>  <VersionBlock firstVersion="1.9"> Optional </VersionBlock>          |

> | `unique_key`    | A tuple of column names that uniquely identify rows.                                                                      | Required. If not provided altered rows will be added twice to the incremental table. |

Author: ```suggestion
| `unique_key`    | A tuple of column names that uniquely identify rows.                                                                      | Required. If not provided altered rows will be added twice to the snapshot |
```

> | `unique_key`    | A tuple of column names that uniquely identify rows.                                                                      | Required. If not provided altered rows will be added twice to the incremental table. |

Reviewer: Incremental table? This is about snapshots not incremental models..

Author: Good catch! It looks like this page could use some larger love. I'll make a note to go back and review a lot of the community adapter pages
